packages/integration-karma/test/component/dynamic-imports/import-config.spec.js

```diff
@@ -0,0 +1,48 @@
+import { createElement } from 'lwc';
+import { registerForLoad, clearRegister } from 'test-utils';
+import ConfigOne from 'x/configone';
+import ConfigTwo from 'x/configtwo';
+
+import PropContainer from 'x/props';
+
+beforeEach(() => {
+    clearRegister();
+});
+
+const macroTask = () => new Promise((resolve) => setTimeout(resolve));
+
+it('should pass props passed as config', async () => {
```

Comment:
I would add a test to make sure the custom element is the same if the `constructor` property never changes but `props` changes.